src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/config/HmppsExternalUsersApiExceptionHandler.kt

```diff
@@ -117,6 +119,20 @@ class HmppsExternalUsersApiExceptionHandler {
       )
   }
 
+  @ExceptionHandler(HttpMessageConversionException::class)
```

Comment:
The additional exceptionhandler was needed to handle the exception thrown - it's not just a basic validationexception